third_party/stdlib/dummy_thread.py

```diff
@@ -0,0 +1,145 @@
+"""Drop-in replacement for the thread module.
```

Comment:
thread.py is already in grumy, I was just not sure what's that lock function does.